.github/workflows/protocol-devchain-anvil.yml

```diff
@@ -124,17 +123,35 @@ jobs:
       - name: 'Install packages'
         shell: bash
         run: yarn
-
-      - name: Generate migrations and run devchain
+          
+      # Install `lsof` dependency, because it's not readily available on CI, but is required by 
+      # `create_and_migrate_anvil_l2_devchain.sh`, because it uses `stop_anvil.sh` to kill 
+      # existing anvil servers.
+      - name: Generate L1 and L2 migrations
         if: success() || failure()
-        run: ./scripts/foundry/create_and_migrate_anvil_devchain.sh
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y lsof
+          
+          ./scripts/foundry/create_and_migrate_anvil_l2_devchain.sh
 
-      - name: Run integration tests against local anvil devchain
+      - name: Run integration tests against L1 devchain
         if: success() || failure()
         run: |
+          source ./scripts/foundry/constants.sh
+
+          anvil \
+          --port 8547 \
```

Comment:
There is a bug on CI:

```
2024-07-08T09:46:34.824960Z ERROR foundry_evm_core::fork::init: It looks like you're trying to fork from an older block with a non-archive node which is not supported. Please try to change your RPC url to an archive node if the issue persists.
Error: 
Could not instantiate forked environment with fork url: http://127.0.0.1:8547/
Context:
- Failed to get block for block number: 256
latest block number: 256
```

Source: [CI](https://github.com/celo-org/celo-monorepo/actions/runs/9837180487/job/27154422476?pr=11107#step:20:599)